data/migrations/20181011141100_shopping_list_view.js

```diff
@@ -0,0 +1,7 @@
+exports.up = function(knex, Promise) {
+    return knex.raw('create view shopping_lists as SELECT recipes.name AS Recipe, ingredients.name AS Ingredient, recipe_ingredients.amount AS Amount, recipe_ingredients.unit AS Type FROM recipe_ingredients INNER JOIN ingredients ON ingredients.id = recipe_ingredients.ingredient INNER JOIN recipes ON recipes.id = recipe_ingredients.recipe');
+};
```

Comment:
Nice job implementing a view. I've never learned this myself so it is nice to see an example :D